src/Form/Option.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\JsonSchema\FormBuilder\Form;
+
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
+
+class Option
+{
+    public function __construct(
+        public string $key,
+        public ?string $label = null,
```

Comment:
Shall we make the label mandatory? For translations is not necessary, but I think that there is value of enforcing a label - makes the forms readable at minimum!